src/pyEQL/solution.py

```diff
@@ -2699,3 +2699,33 @@ def add_solvent(self, formula: str, amount: str):  # pragma: no cover
         mw = self.get_property(formula, "molecular_weight")
         target_mol = quantity.to("moles", "chem", mw=mw, volume=self.volume, solvent_mass=self.solvent_mass)
         self.components[formula] = target_mol.to("moles").magnitude
+
+    def to_phreeqc(self, units: str = "mol/L", charge_balance: bool = "False") -> str:
+        """Generate a PHREEQC input string representing the Solution.
+
+        Returns:
+            A string that can be used as part of a PHREEQC input file to define
+            the solution.
+        """
+        lines = ["SOLUTION 0"]
+        lines.append(f" temp {self.temperature.to('degC').magnitude:.2f}")
```

Comment:
`Solution` also has `pressure` and `volume` attributes. How does PHREEQC handle those? If possible, pass them into the input file, too